### PR TITLE
272006 - Order breeds by id

### PR DESCRIPTION
**Pull request by sfitz42**

### CODE REVIEW

**Review by jbarnsley10 — APPROVED** [review]
LGTM

File: app/config/db.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const { Sequelize } = require('sequelize')
 const { DefaultAzureCredential } = require('@azure/identity')
+const { schema } = require('../schema/people')
```
Comment (jbarnsley10) [inline app/config/db.js:3]: Not sure this import is needed

File: app/config/db.js [hunk 2]
```diff
@@ -48,6 +49,8 @@ const dbConfig = {
   }
 }
 
+console.log(dbConfig.schema)
```
Comment (jbarnsley10) [inline app/config/db.js:52]: Are you intending to output the schema value? If so, perhaps add the word 'schema' in your output

Comment (sfitz42) [reply app/config/db.js]: Removed - Just added in to test it was using the right schema.

**Pull request merged**